### ISSUE TITLE
GGMAX Enhancement - User Writable LUT Folder

### ISSUE THREAD

**Issue by Necrym59** [TheGameCreators/GameGuruRepo#5637]
LUT's can be changed via behaviors (eg: lut_ zone and lut_changer), at present in each of those behaviors it is selecting from the max install system "editors\lut\".   The LUT or LUT Group is selected within the behaviors.
![luts](https://github.com/user-attachments/assets/8eed2cdc-156e-4cc0-85f2-65c48752bb47)

As users can now also create their own LUT's we now need a copy of the "editors\lut\" (including existing default LUT's) created in the users project folder at project creation and any calls to a LUT's can be selected from there instead by default. This allows for new individual LUT's to be created there and even transition effect LUT's in subfolders (eg: "editors\lut\effects\colortosepia\" for example). It also allows for the existing LUT's to be used and transitioned from/to.

Graphix xLUT tool generates user LUT's and effects and he will adjust his program to generate them into the above a project user writable folder and sub folders accordingly.

**Comment by GraPhiX-Guru** [comment]
Yes this as my full support, the LUT files need a little love the text file used complicates LUT selection is this text file really needed?
here is a video using LUTs and Necrym59's new behaviour
https://www.youtube.com/watch?v=r0ZJt_01YmU
